scripts/mbedtls_dev/ecp.py

```diff
@@ -76,6 +76,55 @@ def result(self) -> List[str]:
     def is_valid(self) -> bool:
         return True
 
+class EcpP224R1Raw(bignum_common.ModOperationCommon,
```

Comment:
Minor, style: we normally have two empty lines before and after class definitions.